internal/operations/check/operation.go

```diff
@@ -88,6 +88,12 @@ func (o *Operation) Behave(ctx context.Context, in models.CmdCheckIn) (models.Cm
 				Used: spec.Allow.DeepScan.Value == true,
 				Hint: "switch 'allow.deepScan = true' (or delete) to on",
 			},
+			{
```

Comment:
it should be removed from here, as it's not just displaying options, but rather layers of linting:
- base
- + check vendors
- + check injections (actual code)

Reply:
thanks for the review, fixed and updated .ct files accordingly